html/js/lawyers.js

```diff
@@ -104,7 +104,7 @@ SetupLawyers = function(data) {
                 $("#lawyerid5-" + i).data('LawyerData', lawyer5);
             });
         } else {
-            var element = '<div class="lawyer-list"><div class="no-lawyers">There are no ambulance personnel a available.</div></div>'
+            var element = '<div class="lawyer-list"><div class="no-lawyers">There are no ems available.</div></div>'
```

Comment:
There is no EMS available.

Reply:
Fixed my mistake with a new PR , Appreciate it